python/ml4ir/applications/ranking/tests/test_metrics.py

```diff
@@ -4,64 +4,59 @@
 import os
 import numpy as np
 
-from ml4ir.base.data.ranking_dataset import RankingDataset
-from ml4ir.base.model.ranking_model import RankingModel
+from ml4ir.base.data.relevance_dataset import RelevanceDataset
+from ml4ir.applications.ranking.model.ranking_model import RankingModel
 from ml4ir.base.features.feature_config import FeatureConfig, parse_config
 from ml4ir.applications.ranking.tests.test_base import RankingTestBase
 
 
 # Constants
 GOLD_METRICS = {
-    "old_ACR": 1.6669034,
-    "new_ACR": 2.3998579,
-    "old_MRR": 0.7806604,
-    "new_MRR": 0.6045914,
-    "perc_improv_ACR": -43.9710,
-    "perc_improv_MRR": -22.5538,
+    "old_ACR": 1.6669,
```

Comment:
as discusssed: we still need a better way to do this...  :\

Reply:
Let me file a github issue to track this.

Reply:
I would reach out to Mario, actually, on what his recommendations are for unit tests training ML models.  I tend to go for sanity checks: take a "toy" training dataset where clicks show up relatively randomly (with a simple positional bias) yet where where the "popularity" feature is clearly correlated with clicks.  Compute by hand what the MRR of the positional bias random click would be, and then assert that training for 5 epochs using the popularity feature (and the others) gets strictly better MRR, and that in fact 10 epochs is even better than 5 epochs.

We don't need this for _this_ PR, but we really need more like that kind of testing.